rust-toolchain.toml

```diff
@@ -0,0 +1,13 @@
+[toolchain]
+channel = "1.77.0"
```

Comment:
This is intentionally one version older than the most recent stable version to check that Renovate works and creates a new PR to update the version next day.